user/storage.md

```diff
@@ -0,0 +1,266 @@
+access to your own personal repositories, but might need approval from admins of GitHub
+organizations if you want to push to repos in that organization.
+
+## Cloud Object Storage
+
+Your hub lives in the cloud.
+The preferred way to store data in the cloud is using [cloud object storage](https://aws.amazon.com/what-is-cloud-object-storage/), such as Amazon S3 or Google Cloud Storage.
+Cloud object storage is essentially a key/value storage system.
+They keys are strings, and the values are bytes of data.
+Data is read and written using HTTP calls.
+
+The performance of object storage is very different from file storage.
+On one hand, each individual read / write to object storage has a high overhead (10-100 ms), since it has to go over the network.
+On the other hand, object storage “scales out” nearly infinitely, meaning that we can make hundreds, thousands, or millions of concurrent reads / writes.
+This makes object storage well suited for distributed data analytics.
+However, data analysis software must be adapted to take advantage of these properties.
+
+### Cloud-Native Formats
+
+Cloud-native file formats are formats that are designed from the beginning to
+work well with cloud object storage.
+These formats permit exploration of data and metadata without downloading of the
+entire file / dataset and work well with distributed parallel computing.
+Here we enumerate some popular cloud-native formats and their use cases:
+
+| Format | Use Case | Python Libraries |
+|--|--|--|
+| [Apache Parquet](https://parquet.apache.org/) | Column-oriented data file format designed for efficient data storage and retrieval. Suitable for tabular-style data (rows and columns). | pandas, dask.dataframe, vaex, pyarrow |
+| [Zarr](http://zarr.dev/) | Storage of large multidimensional arrays | zarr, numpy, dask.array, xarray |
+| [Cloud Optimized Geotiff](https://www.cogeo.org/) | Geospatial raster data | rasterio, rio-xarray |
+
+There are other more specialized cloud-optimized formats for specific scientific domains.
+
+It is recommended to use cloud-native formats when working with big data in cloud object storage.
+
+### Working with Object Storage
+
+From a user perspective, the main challenge of working with object storage is the need
+to use more specialized tools, rather than just simple files / filenames, to manage data.
+Fortunately, excellent tools exist to make working with object storage easy and familiar.
+
+For python users, the main tool is [filesystem spec](https://filesystem-spec.readthedocs.io/en/latest/)
+(fsspec), a set of packages which enable us to work with many different types of storage.
+Separate fsspec packages exist for each type of object storage:
+
+- **[s3fs](https://s3fs.readthedocs.io/en/latest/)** - for working with AWS S3
+  (Simple Storage Service) and compatible APIs. Most third-party object storage
+  services (e.g. [Wasabi](https://wasabi.com/) and [Open Storage Newtork](https://www.openstoragenetwork.org/))
```

Comment:
The link checker is erroneously flagging this as a broken link

```
(    user/storage: line  109) broken    https://www.openstoragenetwork.org/ - HTTPSConnectionPool(host='www.openstoragenetwork.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1131)')))
```

Reply:
I added an exception to openstoragenetwork, not sure why it wasn't working